src/amo/components/LandingPage.scss

```diff
@@ -1,5 +1,58 @@
 @import "~amo/css/inc/vars";
+@import "~core/css/inc/mixins";
 
 .LandingPage {
   padding: $padding-page;
 }
+
+.LandingPage-header {
+  align-items: center;
+  display: flex;
+  flex-direction: column;
+  margin: $padding-page $padding-page ($padding-page * 2);
+}
+
+.LandingPage-header-top {
+  align-items: center;
+  display: flex;
+  justify-content: center;
+  margin-bottom: $padding-page * 2;
+}
+
+.LandingPage-header-text {
+  @include margin-start(20px);
+
+  display: inline-block;
+  max-width: 350px;
+}
+
+.LandingPage-heading {
+  font-size: 14px;
+  margin: 0;
+}
+
+.LandingPage-heading-content {
+  margin: 5px 0 0;
+  font-size: 12px;
+}
+
+.LandingPage-header-bottom {
+  display: flex;
+  width: 100%;
+}
+
+.LandingPage-browse-button,
+.LandingPage-browse-button:link {
+  align-items: center;
+  display: flex;
+  font-size: 14px;
+  font-variant: all-small-caps;
+  justify-content: center;
+  margin: 0 $padding-page;
+  padding: 1em;
```

Comment:
We don't use `em`  most of the time, so this seems out-of-place. I'd rather stick to the same units everywhere, `px` here would be better I think.